test/evergreen.yml

```diff
@@ -546,16 +555,25 @@ functions:
     command: shell.exec
     params:
       working_dir: "wiredtiger/cmake_build/test/format"
+      shell: bash
       script: |
         set -o errexit
         set -o verbose
+        # Fail, showing the configuration file.
+        fail() {
+          echo "======= FAILURE =========="
+          [ -f RUNDIR/CONFIG ] && cat RUNDIR/CONFIG
+          exit 1
+        }
+
         for i in $(seq ${times|1}); do
-          ./t -c ${config|../../../test/format/CONFIG.stress} ${trace_args|-T bulk,txn,retain=50} ${extra_args|} || ( [ -f RUNDIR/CONFIG ] && cat RUNDIR/CONFIG ) 2>&1
+          ./t -c ${config|../../../test/format/CONFIG.stress} ${trace_args|-T bulk,txn,retain=50} ${extra_args|} || fail 2>&1
```

Comment:
Though not directly related to the fix, the trailing `2>&1` feels more useful appending to the `./t` command.

Reply:
True.  Actually, in this case the `2>&1` isn't doing much for us.  It's useful when piping to another command (we're not here) or where the error stream would otherwise be lost.  But evergreen is capturing both output and error and logging them.  So I think I'd remove the `2>&1` it's not really helping (or hurting) anything.

Reply:
Agreed. Removed 2>&1